docs/EN_US/HPCCClientTools/CT_Mods/CT_ESDL_CLI.xml

```diff
@@ -1049,6 +1049,29 @@
           appropriate gateway's connection information, then use that
           information to create the actual gateway request connection.</para>
         </sect3>
+
+        <sect3>
+          <title>Configuring ESDL binding for Proxy Mode methods</title>
+
+          <para>You can specify a proxy for ESDL service methods on another
```

Comment:
Perhaps something like "You can specify that ESDL service methods be proxied to another ESP instance"?